homeassistant/components/fireservicerota/sensor.py

```diff
@@ -0,0 +1,128 @@
+"""Sensor platform for FireServiceRota integration."""
+import logging
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import callback
+from homeassistant.helpers.dispatcher import async_dispatcher_connect
+from homeassistant.helpers.restore_state import RestoreEntity
+from homeassistant.helpers.typing import HomeAssistantType
+
+from .const import DOMAIN as FIRESERVICEROTA_DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(
+    hass: HomeAssistantType, entry: ConfigEntry, async_add_entities
+) -> None:
+    """Set up FireServiceRota sensor based on a config entry."""
+    coordinator = hass.data[FIRESERVICEROTA_DOMAIN][entry.entry_id]
+
+    async_add_entities([IncidentsSensor(coordinator)])
+
+
+class IncidentsSensor(RestoreEntity):
+    """Representation of FireServiceRota incidents sensor."""
+
+    def __init__(self, coordinator):
+        """Initialize."""
+        self._coordinator = coordinator
+        self._entry_id = self._coordinator._entry.entry_id
```

Comment:
We shouldn't access private attributes from outside the instance. Make them non private if needed.

Reply:
Implemented